alvr/server_core/src/tracking/mod.rs

```diff
@@ -435,37 +435,41 @@ pub fn tracking_loop(
         ) {
             let mut hand_gesture_manager_lock = hand_gesture_manager.lock();
 
-            if let Some(hand_skeleton) = tracking.hand_skeletons[0] {
-                ctx.events_sender
-                    .send(ServerCoreEvent::Buttons(
-                        hand_gestures::trigger_hand_gesture_actions(
-                            gestures_button_mapping_manager,
-                            *HAND_LEFT_ID,
-                            &hand_gesture_manager_lock.get_active_gestures(
-                                hand_skeleton,
-                                gestures_config,
+            if !device_motion_keys.contains(&*HAND_LEFT_ID) {
```

Comment:
Fine, but getting rid of a level of indentation would probably be better
So `if let ... = (!device_motion_keys.contains(&*HAND_LEFT_ID)).then_some(tracking.hand_skeletons[0]).flatten()`, but idk

Reply:
Not sure that when formatted it will change anything. But on another note we are veeery close to if-let chains in stable rust (with 2024 edition) so at that point I will do a codebase-wide refactor for that